### PR TITLE
Update shopify_hydrogen.md

### PR DESCRIPTION
**Pull request by tristanlee85**
Expand build command and Node limits

### CODE REVIEW

File: src/pages/guides/shopify_hydrogen.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ You’ve installed the following dependencies:
 
 [Node.js](https://nodejs.org/en/) version 16.5.0 or higher
 
+_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version] for more information.
```
Comment (tristanlee85) [inline src/pages/guides/shopify_hydrogen.md:23]:
```suggestion
_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) for more information.
```

File: src/pages/guides/shopify_hydrogen.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ You’ve installed the following dependencies:
 
 [Node.js](https://nodejs.org/en/) version 16.5.0 or higher
 
+_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) for more information.
```
Comment (tristanlee85) [inline src/pages/guides/shopify_hydrogen.md:23]:
```suggestion
_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
```

**Comment by rishi-raj-jain** [comment]
@tristanlee85 

It shall be a callout instead of a note, right?

**Comment by rishi-raj-jain** [comment]
<Callout type="info">
{{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
</Callout>

**Comment by rishi-raj-jain** [comment]
Adding node build command was a new one for me!
Create a PR on the example to let the users switch between the two when not in the workflow: https://github.com/layer0-docs/layer0-shopify-hydrogen-example/pull/60

File: src/pages/guides/shopify_hydrogen.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ You’ve installed the following dependencies:
 
 [Node.js](https://nodejs.org/en/) version 16.5.0 or higher
 
+_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
```
Comment (tristanlee85) [inline src/pages/guides/shopify_hydrogen.md:23]:
```suggestion
<Callout type="info">
{{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
</Callout>
```

**Comment by rishi-raj-jain** [comment]
@tristanlee85 

Seems like markdown doesn't work inside the callout: 
Screenshot 2022-07-01 at 7 22 16 PM.

**Pull request merged**